brainstormforce/ultimate-addons-for-gutenberg
# Improvement: UI/UX Changes suggested by Monika

### Description

- Team - https://ua-gutenberg.atlassian.net/browse/UAG-401
- WP-Search - https://ua-gutenberg.atlassian.net/browse/UAG-426
- Lottie - https://ua-gutenberg.atlassian.net/browse/UAG-404
- Taxonomy List - https://ua-gutenberg.atlassian.net/browse/UAG-400
- Table of Content - https://ua-gutenberg.atlassian.net/browse/UAG-369
- Review Schema - https://ua-gutenberg.atlassian.net/browse/UAG-384
- FAQ Schema - https://ua-gutenberg.atlassian.net/browse/UAG-372
- Post timeline - https://ua-gutenberg.atlassian.net/browse/UAG-379
- Content Timeline - https://ua-gutenberg.atlassian.net/browse/UAG-380
- CF7 - https://ua-gutenberg.atlassian.net/browse/UAG-429
- Multi Buttons - https://ua-gutenberg.atlassian.net/browse/UAG-383
- Star - https://ua-gutenberg.atlassian.net/browse/UAG-424
- Inline Notice - https://ua-gutenberg.atlassian.net/browse/UAG-394
- Info Box - https://ua-gutenberg.atlassian.net/browse/UAG-396
- Section - https://ua-gutenberg.atlassian.net/browse/UAG-393
- Tabs - https://ua-gutenberg.atlassian.net/browse/UAG-399

### Types of changes




- Improvements

### How has this been tested?

- Drag and drop each block
- Check suggested changes

### Checklist:
